### DISCUSSION TITLE
essay/2023/11/29/welfare-of-company

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#151] [Comments]
# essay/2023/11/29/welfare-of-company

복지가 좋은 회사가 부러웠습니다. 이런저런 혜택이 많은 회사. 우와 좋다. 나도 저런 회사 다녀보고 싶어.

https://jeho.page/essay/2023/11/29/welfare-of-company.html

**Comment by yoonjong-park** [comment]
회사의 모든 제도는 회사의 퍼포먼스 때문이라는 말이 떠오르네요.
https://www.kmis.or.kr/3_sig/sem_data/2010_0320.pdf   
60쪽에 있습니다. 도움이 될까 싶어 링크 남깁니다.
